README.md

```diff
@@ -251,9 +251,9 @@ A directory that contains license texts which are not provided by ORT.
 
 A Kotlin script that enables the injection of how-to-fix texts in Markdown format for ORT issues into the reports.
```

Comment:
> Using the sub tool name as prefix seems to work only as long as the file is consumed by just that tool.

True. But in lack of a better suggestion of a prefix, I believe using the name of the only tool that interprets the file is fine for now.

> It adds boilerplate, file names before seemed to have enough information.

I'm afraid that we're simply without any alternative if we want to get auto-completion in how-to-fix-hints scripts right, which IMO is a must-have.

> So, if we should decide to change the filenames to double extensions what do you think about how-to-fix-text-provider.ort.kts instead.

I basically like that proposal (and I was thinking of doing something similar for ORT result files, too, and name them e.g. "analyzer.ort.yml", as that would allow to associate these file extensions with a program like the ORT Workbench), but in case of scripts it would not work: [Kotlin script require a dedicated `fileExtension`](https://github.com/oss-review-toolkit/ort/blob/eb88bd776c0accb75f3741aaf398d858c4428739/reporter/src/main/kotlin/HowToFixTextProviderScriptTemplate.kt#L31) that is associated to their respective type of scripts only.

> Apart from that, do you happen to have a reference why "double extension" is needed?

See above.

> Is there an alternative?

Not that I'm aware of.

Reply:
> I'd prefer to have this approved by an additional reviewer.

@mnonnenmacher or @MarcelBochtler, can you confirm this is also fine with you, incl. the required file renaming in your configuration?